components/multi_page/view.coffee

```diff
@@ -10,11 +10,11 @@ module.exports = class MultiPageView extends Backbone.View
   events:
     'click .js-page': 'change'
 
-  initialize: ({ @title, @description, @pages }) ->
+  initialize: ({ @title, @description, @pages, @defaultPage }) ->
     @collection = new Backbone.Collection _.map @pages, (id, title) ->
       new Page id: id, title: title
 
-    @state = new Backbone.Model active: @collection.first().id
+    @state = new Backbone.Model active: if @defaultPage then @defaultPage else @collection.first().id
```

Comment:
Wondering if I'm being too presumptuous that the default page id passed is actually within the collection? Should I check that `@defaultPage` has a value and that it's included in the collection with the `findWhere` method or something?

Also thinking I should probably change `@defaultPage` to `@defaultPageId` to be a little more clear. 

Let me know your thoughts?